### ISSUE TITLE
Antibiotics and wounds for bionic surgery

### ISSUE THREAD

**Issue by worm-girl** [CleverRaven/Cataclysm-DDA#74195]
### Is your feature request related to a problem? Please describe.

When you have surgery IRL, you come away very injured and with a course of antibiotics to prevent infection. CDDA is using future tech with presumably minimally invasive techniques, but even laproscopic surgeries IRL are not something you wake up from feeling 100%.

### Solution you would like.

Getting any operation, no matter how successful it is, should debuff your lifestyle score and do some damage to your body. There should also be a risk of an infection developing if the patient does not take a course of antibiotics. These could be given as a depot at the time of surgery (perhaps supplied to the surgeon ahead of time), or it could be a thing you have to take pills for for a few days after. The former would be easier on the player. As a cautionary measure, the doctor or autodoc should assess the patient and warn them if they're too weak or injured to survive an operation.

One argument against something like this is that installing little parts like power storage units often requires multiple operations, but if the damage and infection risk scale with the body part and bionic slot costs, then it will be safe enough to get multiple things installed in one trip, you'll just have more downtime afterwards.

### Describe alternatives you have considered.

I can't think of a good reason not to do this. It's a little weird that you can have titanium rods bolted to your skeleton, or a chunk of your brain scooped out and replaced with a computer, and wake up eight hours later feeling like a million bucks. There also aren't enough uses for antibiotics, which are an incredibly important item IRL.

This could all be waived in IE Aftershock, where the tech is more advanced and it's expected you'll be swapping out parts.

### Additional context

dead space 2 eye surgery machine dot gif

**Comment by IdleSol** [comment]
We should not forget about the Exodii. As I understand it, this is now the main way to get bionics and install them.

**Comment by anoobindisguise** [comment]
I agree that the big problem is that bionics are installed one-by-one. You should go to rubik and hand over like ten cbms for one (long) session of surgery, and an associated week or so of recovery or whatever. It's rather silly that currently they put you under, then open you up, shove an AA battery next to your pancreas, and then suture the wound, let the anesthetic haze wear off, and then turn you loose until you want another bionic (which is usually like. 5 seconds later)

**Comment by Mithrite** [comment]
Just compiling thoughts:
- damage on every surgery for affected bodypart(s)
- variable based on roll, even for success
- likewise for infection chance (pockets for antibiotics and anesthesia in autodocs would be nice, but I don't think multiple pockets in furn./tiles is supported, unless you treat it like the autoclave I suppose)
- health damage
- batch operations
- stamina/speed reduction or the like as anesthesia wears off
- longer term itchyness at bionic site as it heals
- mutation "disabling" by surgical removal for certain addons like photophore, potentially reverted over time by MEND_ALL effects
- wounds system integration obviously, once that's further along
- variable results based on patient health and limb traits at time of surgery
- variable results/etc based on surgery scale
